module/VuFind/tests/unit-tests/src/VuFindTest/I18n/Locale/LocaleDetectorFactoryTest.php

```diff
@@ -59,6 +59,7 @@ protected function getStrategyClasses()
     {
         $factory = new LocaleDetectorFactory();
         $strategies = $this->callMethod($factory, 'getStrategies', func_get_args());
+        $strategyClasses = [];
```

Comment:
How about reducing this to `return array_map('get_class', $strategies);`?

Reply:
I am always for simplifying code. Done (with little help of `iterator_to_array`).